### DISCUSSION TITLE
What happened to package "@graphql-codegen/relay-operation-optimizer"

### DISCUSSION THREAD

**Discussion by Woodz** [dotansimha/graphql-code-generator#9323] [Q&A]
I really would like to use this package to solve a problem I have, but the most recent package in npm is 1.11.2 from 3 years ago which is not compatible with latest stable version of `graphql-code-generator` and `graphql`. I see a lot of documentation and referenced in this repo to this plugin, but looks like it has been abandoned.

Can anyone share what happened and whether there are any plans to revive it?

**Comment by ardatan** [comment]
Because it is not needed anymore as a seperate package. It is available under `flattenGeneratedTypes: true` flag;
https://the-guild.dev/graphql/codegen/plugins/typescript/relay-operation-optimizer#usage

**Reply by Woodz** [reply]
Hmm, I'm trying to inline the fragments in my query `DocumentNode`s that are generated by the `typescript-document-nodes` plugin, but this does not appear to be supported.

It looks like `flattenGeneratedTypes` only works when generating TS types, not `DocumentNodes` - hence why I was looking at `@graphql-codegen/relay-operation-optimizer` package because `@graphql-tools/relay-operation-optimizer` supports inline fragments in `DocumentNode`s
